src/index.js

```diff
@@ -1,8 +1,11 @@
-import getClasses from './getClasses';
-
-console.log('ran from index.js');
-getClasses();
-
-const obj = {a: 'alpha',b: 'bravo'};
-const newObj = {...obj, c: 'charlie'}
-console.log(newObj)
\ No newline at end of file
+// import html from './index.html';
+import './style.css';
+
+window.addEventListener('load', () => {
```

Comment:
**Kindly use ES6 modules to write modular JavaScript with proper ES6 syntax.**

- usage of arrow functions instead of the function keyword.
- usage of import and export instead of require or module.exports.
- proper usage of const and let keywords.

(Requested in the last point of your requirements)
![Screen Shot 2021-08-02 at 13 13 57](https://user-images.githubusercontent.com/22258274/127905626-d0766da9-7891-4c79-8746-2c082640e00f.png)